src/components/calcite-link/readme.md

```diff
@@ -32,6 +32,12 @@ You can programmatically focus a `calcite-link` with the `setFocus()` method:
 
 Type: `Promise<void>`
 
+## Slots
+
+| Slot                                | Description |
+| ----------------------------------- | ----------- |
+| `"default text slot for link text"` |             |
```

Comment:
@benelan this one needs a name or dash I think.

Reply:
This one I didn't add myself, I just moved it. It was just in the wrong location so the slot wasn't being added to the generated html. It sounds like it is a default slot, should it be named "default"?

Reply:
Gotcha. yeah it was already incorrect. Can we fix it since it moved? I think for default slots we usually just add a dash.

Like this:

```
 * @slot - A slot for adding `calcite-action`s that will appear at the top of the action bar.
 ```

Reply:
> Approved after slot fix. We should discuss if we always should document default slot. My opinion is that we should document it and what a user should slot there.

Got it thanks. I'll take a yay/nay count for documenting default slots in our next team meeting.